Makefile

```diff
@@ -1,56 +1,63 @@
 .PHONY: cache composer-install composer-update create-network help install jwt up update upAll
 
 CONFIG_DIR=api/config
-DC=docker-compose
-DC_UP=$(DC) up -d
-DC_EXEC=$(DC) exec php
-BIN_CONSOLE=$(DC_EXEC) bin/console
```

Comment:
Because I consider it being unnecessary complicated. Why would you declare variables in the first place ? When I had to debug the installation I had to manually read each variables which was a pain in the ass to begin with.
Also using these variables prevented me from adding arguments. For example the composer-update I had to add the -e option twice which is not possible if I was using the variables you declared since the -e option must be located between exec and php.